vercel.json

```diff
@@ -4,5 +4,6 @@
             "source": "/(.*)",
             "destination": "/"
         }
-    ]
+    ],
+    "installCommand": "npm install --legacy-peer-deps"
```

Comment:
Why do we need this now? :thinking:

Reply:
Because we're using outdated versions (for something I forgot, I think node) with newer packages (such as 'ai')